src/sql/parts/dashboard/dashboardWebviewTab.component.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the Source EULA. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { Component, forwardRef, Input, OnInit, Inject, ChangeDetectorRef, ElementRef } from '@angular/core';
+
+import Event, { Emitter } from 'vs/base/common/event';
+import Webview from 'vs/workbench/parts/html/browser/webview';
+import { Parts } from 'vs/workbench/services/part/common/partService';
+import { IDisposable } from 'vs/base/common/lifecycle';
+
+import { DashboardTab } from 'sql/parts/dashboard/common/interfaces';
+import { TabConfig } from 'sql/parts/dashboard/common/dashboardWidget';
+import { DashboardServiceInterface } from 'sql/parts/dashboard/services/dashboardServiceInterface.service';
+import { IDashboardWebview } from 'sql/services/dashboardWebview/common/dashboardWebviewService';
+
+@Component({
+	template: '<div></div>',
+	selector: 'dashboard-webview-tab',
+	providers: [{ provide: DashboardTab, useExisting: forwardRef(() => DashboardWebviewTab) }]
+})
+export class DashboardWebviewTab extends DashboardTab implements OnInit, IDashboardWebview {
```

Comment:
>DashboardWebviewTab [](start = 13, length = 19)

Should we move this inside the common folder? It should be the same folder is dashboardWidgetTab.Component

Reply:
It's not realyl a common component. Its a stand alone component so I think actually the dashboardwidget tab should be pulled out.